docs/quick-ref.md

```diff
@@ -0,0 +1,148 @@
+# Quick Reference
+
+!!! note "notice !!!!"
+    (page under construction!!!)
+
+<!--- TODO: Should we use collapsible admonitions to help find what you need without feeling lost, or would that complicate Ctrl+Z too much? things for kyuu to true think about. ---> 
+
+## Basic Rolling
+
+<!--- TODO: I might add the graphical colored version of the Four Actions table I made --->
+
+**Q: Can I Create An Advantage with a Combat Style?**
+Yes, when deemed appropriate.
+
+**Q: How do I do a multi-target roll?**
+1. Do one regular Action roll.
+2. Distribute resulting Shifts among targets. 
+3. Each target can Defend against their incoming Shifts.
+
+**Q: How do I roll against a full zone?**
+1. Do one regular Action roll. 
+2. Each person in the Zone Defends against that number. 
+
+### Armor & Weapon
+*(See: [Extra Rules page](adopted-rules))*
+
+* Apply only when the Attack is successful. 
+* Does not count toward Succeed with Style.
+
+**Weapon**
+* Add Weapon Level to the total damage dealt. 
+* On a Tie, deal Weapon Level as damage instead of gaining a Boost
+
+**Defend**
+* Subtract Armor Level from the total damage received.
+* If Armor reduces damage to 0, Defender gains a Boost. 
+
+## Fate Points
+
+Fate Points can be spent to:
+1. Invoke an Aspect
+2. Compel an Aspect
+3. Declare a Story Detail
+4. Power a Stunt
+
+Fate Points can be earned by:
+1. Accepting a Compel
+2. Hostile Invokes on your Character/Consequence Aspects, paid with Fate Points
+3. Concede from a conflict
+
+**Q: If my refresh is full, can I still earn FP?**
+Yes!
+
+**Q: Session ended in the middle of combat. Do I get my FP back?**
+No! 
+
+**Q: Does it cost anything to refuse a Compel?**
+No; you are already losing out on the offered Fate Point. 
+
+**Q: How many FP does the GM get?**
+One for each PC in the scene, shared by all NPCs.
+
+## Aspects & Command Seals
+
+**Q: What's a Boost?**
+A single-use Aspect, always free to invoke.
+
+**Q: How many Free Invokes can I use at once?**
+As many as you want. 
+
+**Q: Can I use Free Invokes from different Aspects at the same time?**
+Yes
+
+**Q: Can I use a Fate Point with Free Invokes?**
+Yes, but only one.
```

Comment:
We should separate all of the Q&A to the end of the quickref, so it's easier to parse the diagrams and rules at a glance

Reply:
What do you think of making them collapsible boxes under their appropriate headers with Admonitions? My concern with moving them to the bottom is that players might not look for them under there. Unless you mean that all Diagrams should be figures just at the top standalone?

Reply:
We could make them collapsible admonitions, that'd be fine. I'm also fine with them being on another page titled "Q&A" or on a lower section called "Frequent Q&A"

If you're worried people won't see them, remember that there will be a TOC to their right that will list all of the sections, so Q&A should be visible.

Reply:
I'm mostly just concerned that if the quick ref is too filled with text it will be hard to reference quickly 🙂

Reply:
I wouldn't say it's so much "worried people won't find them" and more worry about bouncing them around the page for an extra step when the heading implies that your answers about that thing would be found beneath it.

Since my solution involves the admonitions, should we just address this in the next revision? Or should I make Q&A now?